db/readme.md

```diff
@@ -0,0 +1,105 @@
+# Import Directories
+
+## What is the import directory for?
+
+The `import/` directory provides a way for you to change your config settings without the need to even touch the main `/conf/` and `/db/` files.
+
+By placing your custom entries into the `import/` directory within these two locations, your core files will not need to have any conflicts resolved when you update your server. You store your changes, and the rest is updated with rAthena.
+
+## How does this work?
+
+Think of "import" as in "override". Place only the settings you have changed in the import files, or settings you are "overriding".
+
+For example, when setting up a server there are always a few config settings that users would like to change in order for rAthena to suit their needs. The following example will show you how to use the `/db/import/` directory correctly. (for `/conf/import/` eamples, see [/conf/readme.md](/conf/readme.md))
+
+### Achievements
+---
+We want to add our own custom achievement that can be given to a player via an NPC Script and another that we can give to our GMs.
+
+#### /db/import/achievement_db.yml
+
+	Achievements:
+      - ID: 280000
+        Group: "AG_GOAL_ACHIEVE"
+        Name: "Emperio"
+        Reward:
+          TitleID: 1035
+        Score: 50
+      - ID: 280001
+        Group: "AG_GOAL_ACHIEVE"
+        Name: "Staff"
+        Reward:
+          TitleID: 1036
+        Score: 50
+
+
+### Instances
+---
+We want to add our own customized Housing Instance.
+
+#### /db/import/instance_db.txt
+
+	// ID,Name,LimitTime,IdleTimeOut,EnterMap,EnterX,EnterY,Map2,Map3,...,Map255
+    35,Home,3600,900,1@home,24,6,2@home,3@home
```

Comment:
Adjust the tab for the comment structure line to spaces or the instance structure line to a tab to match.